### PR TITLE
[MIG][16.0]account_financial_risk

### PR DESCRIPTION
**Pull request by Vandan-OSI**

### CODE REVIEW

**Review by stefan-tecnativa — APPROVED** [review]
Functional review 👍️ @chienandalu can you take a look at it too?

**Review by sergio-teruel — APPROVED** [review]

File: account_financial_risk/security/security.xml
Comment (pedrobaeza) [inline account_financial_risk/security/security.xml]: You mustn't remove this from XML (the same for all removals of this).

File: account_financial_risk/tests/test_account_financial_risk.py
Comment (pedrobaeza) [inline account_financial_risk/tests/test_account_financial_risk.py:19]: Why activating both currencies?

File: account_financial_risk/views/res_partner_view.xml
Comment (pedrobaeza) [inline account_financial_risk/views/res_partner_view.xml]: You have to put this group on the view elements instead, not simply remove it.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by dreispt** [comment]
Ping @Vandan-OSI

**Comment by BT-anieto** [comment]
Hello @Vandan-OSI ! :) Could you take a look at the last comments to unblock other PRs depending on this one? Thx a lot.

**Comment by Vandan-OSI** [comment]
> Hello @Vandan-OSI ! :) Could you take a look at the last comments to unblock other PRs depending on this one? Thx a lot.

Yes I Will Look at into give me 1 Day

File: account_financial_risk/views/res_partner_view.xml
Comment (gdgellatly) [reply account_financial_risk/views/res_partner_view.xml]: I wonder if this isn't for some debate. We typically remove this group anyway from view, replacing with other groups, because more than an accountant needs to be able to see limits, and risk allow_edit takes care of the rest. Also, in larger companies, not all accountants can edit credit limits, and not everyone who can edit credit limits is an accountant, so we have a completely different security group to do this.

Right now that is a single hidden group, but I could easily make a dedicated Credit Control group with 3 levels say User, Approver, Manager - user is readonly access and the group for the page in this view, approver also readonly here but can approve (currently group_overpass_partner_risk_exception) and manager can approve and set limits. Manager is implied for accountant. What do you think?

**Comment by BT-anieto** [comment]
@Vandan-OSI Hi Vandan! If you don't have time, please tell me so that I can take it.

File: account_financial_risk/tests/test_account_financial_risk.py
Comment (Vandan-OSI) [reply account_financial_risk/tests/test_account_financial_risk.py:19]: @pedrobaeza There is One Method test_invoice_risk_draft_different_currency will Test invoice with Different  currency

**Comment by Vandan-OSI** [comment]
> @Vandan-OSI Hi Vandan! If you don't have time, please tell me so that I can take it.

Done, Thanks for your patience and understanding.

**Comment by pedrobaeza** [comment]
Can you please check tests, which are red right now?

**Comment by BT-anieto** [comment]
@pedrobaeza Hello Pedro. I would be great if you could review this PR, since it is blocking sale_financial_risk. Thank you.

File: account_financial_risk/views/res_partner_view.xml
Comment (pedrobaeza) [reply account_financial_risk/views/res_partner_view.xml]: Once merged, you can improve the module adding more granularity through levels for having this visible as readonly, and another allowing to edit.

File: account_financial_risk/wizards/partner_risk_exceeded_view.xml
Comment (pedrobaeza) [inline account_financial_risk/wizards/partner_risk_exceeded_view.xml]: Don't remove this line. Why are you doing this?

File: account_financial_risk/tests/test_account_financial_risk.py
Comment (pedrobaeza) [inline account_financial_risk/tests/test_account_financial_risk.py:20]: Remove empty line inside of a method.

**Review by FrancoMaxime — COMMENTED** [review]
same comments as @pedrobaeza

**Comment by pedrobaeza** [comment]
Superseded by #292

**Pull request closed without merging**
